index.bs

```diff
@@ -385,6 +414,115 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
         {{VideoConfiguration/contentType}}, if possible. Otherwise, the UA should
         presume that the video stream cannot produce alpha channel information.
       </p>
+
+      <p>
+        <pre class='idl'>
+          enum HdrMetadataType {
+            "smpteSt2086",
+            "smpteSt2094-10",
+            "smpteSt2094-40"
+          };
+        </pre>
+
+        <p>
+          If specified, the <dfn for='VideoConfiguration' dict-member>hdrMetadataType</dfn> 
+          represents the HDR metadata type the decoder needs to support in order
+          to successfully decode the video track.
+        </p>
+
+        <p>
+          The {{HdrMetadataType}} values are:
+          <ul>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2086</dfn>, representing
+              the static metadata type defined by [[SMPTE-ST-2086]].
+            </li>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2094-10</dfn>, representing
+              the dynamic metadata type defined by [[SMPTE-ST-2094]].
+            </li>
+            <li>
+              <dfn for='HdrMetadataType' enum-value>smpteSt2094-40</dfn>, representing
+              the dynamic metadata type defined by [[SMPTE-ST-2094]].
+            </li>
+          </ul>
+        </p>
+      </p>
+
+      <p>
+	      <pre class='idl'>
+	        enum ColorGamut {
+	          "srgb",
+	          "p3",
+	          "rec2020"
+	        };
+	      </pre>
+	
+	      <p>
+	        The {{ColorGamut}} represents the color gamut supported by the UA and
+          output device. That is, if the UA receives content with colors in the
+          specified space it can cause the output device to render the appropriate
+          color, or something appropriately close enough. Note: this definition
+          is borrowed from the 'color-gamut' CSS media query definition.
```

Comment:
add a link to the color-gamut propdef in the CSSWG

Reply:
Good point. I have fixed the link.